src/Types/Resources.ts

```diff
@@ -0,0 +1,27 @@
+export interface Resources {
+  label: string; // Name to be displayed to the user
+  name: string; // unique id for reference
+  description: string; // '' for null
+  tags: string[]; // [] for none
+  dos: string[]; // [] for none
+  donts: string[]; // [] for none
+  liveClip: string; // naration video
+  liveAudio: string; // naration audio handle
+  queAudio: string; // Audio to be played as a que
+  eduClip: string[]; // clip id, '' if null
```

Comment:
Is there a reason you want the empty string for null? Why can't it just be null? This applies anywhere that you want a specific value to represent null. The danger of designating these values to mean null is that they require tribal knowledge in the future
```
eduClip?: string[];
```

Reply:
previewClips